content/index.pl.md

```diff
@@ -0,0 +1,67 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after #.
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "Omówienie podstaw dostępności"
+nav_title: "Omówienie" 
+
+lang: pl   # Change "en" to the translated-language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+last_updated: 2024-04-05   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+translators:
+ - name: "Stefan Wajda" 
+
+github:
+  repository: w3c/wai-fundamentals-overview
+  path: content/index.pl.md    # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+permalink: /fundamentals/pl   # Add the language shortcode to the end, with no slash at end, for example: /fundamentals/fr
+
+ref: /fundamentals/   # Translators, do not change this
+changelog: /fundamentals/changelog/
+feedbackmail: wai@w3.org
+
+# In the footer below:
+# Do not translate or change CHANGELOG or ACKNOWLEDGEMENTS.
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+# Do not change the dates in the footer below.
+footer: >
+  <p><strong>Data:</strong> Aktualizacja 8 sierpnia 2023. DZIENNIK ZMIAN.</p>
+  <p><strong>Redaktor:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Opracowano przy wsparciu Grupy Roboczej ds. Edukacji i Promocji (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+
+inline_css: |
+  img.icon {--img-width:1em}
+  
+---
+
+Poniższe materiały zawierają informacje ułatwiające **rozpoczęcie pracy z dostępnością**:
+
+[[Wprowadzenie do dostępności internetowej]](/fundamentals/accessibility-intro/)
+:   Przedstawia dostępność internetową i łącza do dodatkowych materiałów. Obejmuje dostępność w kontekście; dlaczego dostępność jest ważna dla osób indywidualnych, firm i społeczeństwa; udostępnianie technologii cyfrowej; ocenę dostępności; i  proste przykłady.
+    -   [Film Wprowadzenie do dostępności internetowej i standardów W3C](/videos/standards-and-benefits/) 4 minuty {% include image.html src="video.svg" alt="" class="icon" %}
+
+[[Dostępność: Chodzi o ludzi]](/people/)
```

Comment:
```suggestion
[[Dostępność jest dla ludzi]](/people/)
```

Reply:
```suggestion
[[Dostępność: Jest dla ludzi]](/people/)
```
I think the structure of the title needs to be preserved in this case as well. The use of a colon here signals the importance of the following statement

Reply:
"dostępność" does't work with "o" IMHO.
```suggestion
[[Dostępność jest dla ludzi]](/people/)
```
I don't think we need to copy the English interpunction. It does not bring the same semantics or ornament when written in polish language.